tests/test_update_manager_specific_lines.py

```
@@ -0,0 +1,238 @@
+    def temp_dir(self) -> Generator[Path, None, None]:
+        """Create temporary directory for tests."""
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            yield Path(tmp_dir)
+
+    @pytest.fixture
+    def manager(self, temp_dir: Path) -> DatabaseUpdateManager:
+        """Create DatabaseUpdateManager instance."""
+        return DatabaseUpdateManager(cache_dir=temp_dir, update_interval_hours=24)
+
+    @pytest.mark.asyncio
+    async def test_backup_load_exception_logging_lines_329_330(self, manager, temp_dir):
```

Ruff (0.14.1)

Unused method argument: `temp_dir`

(ARG002)